backend/apps/accounts/views.py

```
@@ -1105,37 +1110,141 @@ def get(self, request):
-        return Response([])
+        suggestions = User.objects.exclude(id=request.user.id).order_by("-date_joined")[
+            :5
+        ]
+        serializer = UserListSerializer(
+            suggestions, many=True, context={"request": request}
+        )
+        return Response(serializer.data, status=status.HTTP_200_OK)
 
 
 class PublicProfileView(APIView):
     permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)